<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Example test run" duration-ms="34328" started-at="2015-09-14T17:01:22Z" finished-at="2015-09-14T17:01:56Z">
    <groups>
    </groups>
    <test name="Simple Test" duration-ms="34328" started-at="2015-09-14T17:01:22Z" finished-at="2015-09-14T17:01:56Z">
      <class name="org.suyog.digitalriver.tests.VerifyPhoneNumberTests">
        <test-method status="PASS" signature="setUp()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="setUp" is-config="true" duration-ms="2390" started-at="2015-09-14T22:31:22Z" finished-at="2015-09-14T22:31:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="checkCountryCodeForNumber()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="checkCountryCodeForNumber" duration-ms="6846" started-at="2015-09-14T22:31:24Z" description="Validates the country code for a phone number." finished-at="2015-09-14T22:31:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkCountryCodeForNumber -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="tearDown" is-config="true" duration-ms="78" started-at="2015-09-14T22:31:31Z" finished-at="2015-09-14T22:31:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="setUp" is-config="true" duration-ms="1624" started-at="2015-09-14T22:31:31Z" finished-at="2015-09-14T22:31:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="checkIsPossibleNumber()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="checkIsPossibleNumber" duration-ms="4884" started-at="2015-09-14T22:31:33Z" description="Check whether a phone number is a possible number given a number in the form of a string, and the region where the number could be dialed from." finished-at="2015-09-14T22:31:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsPossibleNumber -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="tearDown" is-config="true" duration-ms="69" started-at="2015-09-14T22:31:38Z" finished-at="2015-09-14T22:31:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="setUp" is-config="true" duration-ms="1518" started-at="2015-09-14T22:31:38Z" finished-at="2015-09-14T22:31:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="checkIsValidNumber()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="checkIsValidNumber" duration-ms="4785" started-at="2015-09-14T22:31:39Z" description="Tests whether a phone number matches a valid pattern." finished-at="2015-09-14T22:31:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:288)
	at org.testng.Assert.assertEquals(Assert.java:298)
	at org.suyog.digitalriver.pageobjects.DRCustomerSupportPage.readNumberAndVerifyIsValidNumber(DRCustomerSupportPage.java:63)
	at org.suyog.digitalriver.tests.VerifyPhoneNumberTests.checkIsValidNumber(VerifyPhoneNumberTests.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkIsValidNumber -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="tearDown" is-config="true" duration-ms="71" started-at="2015-09-14T22:31:44Z" finished-at="2015-09-14T22:31:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="setUp" is-config="true" duration-ms="1503" started-at="2015-09-14T22:31:44Z" finished-at="2015-09-14T22:31:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="checkNumberType()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="checkNumberType" duration-ms="4435" started-at="2015-09-14T22:31:46Z" description="Gets the type of a phone number." finished-at="2015-09-14T22:31:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:288)
	at org.testng.Assert.assertEquals(Assert.java:298)
	at org.suyog.digitalriver.pageobjects.DRCustomerSupportPage.readNumberAndVerifyNumberType(DRCustomerSupportPage.java:121)
	at org.suyog.digitalriver.tests.VerifyPhoneNumberTests.checkNumberType(VerifyPhoneNumberTests.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkNumberType -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="tearDown" is-config="true" duration-ms="70" started-at="2015-09-14T22:31:50Z" finished-at="2015-09-14T22:31:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="setUp" is-config="true" duration-ms="1533" started-at="2015-09-14T22:31:50Z" finished-at="2015-09-14T22:31:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="checkOfflineGeocoderLocation()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="checkOfflineGeocoderLocation" duration-ms="4429" started-at="2015-09-14T22:31:52Z" description="Checks the offlibe geocoder location for a phone number." finished-at="2015-09-14T22:31:56Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Germany] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Germany] but found []
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at org.suyog.digitalriver.pageobjects.DRCustomerSupportPage.readNumberAndVerifyOfflineGeocoderLocation(DRCustomerSupportPage.java:96)
	at org.suyog.digitalriver.tests.VerifyPhoneNumberTests.checkOfflineGeocoderLocation(VerifyPhoneNumberTests.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkOfflineGeocoderLocation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:org.suyog.digitalriver.tests.VerifyPhoneNumberTests@6093dd95]" name="tearDown" is-config="true" duration-ms="75" started-at="2015-09-14T22:31:56Z" finished-at="2015-09-14T22:31:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- org.suyog.digitalriver.tests.VerifyPhoneNumberTests -->
    </test> <!-- Simple Test -->
  </suite> <!-- Example test run -->
</testng-results>
